src/main/webapp/edit-event-form.html

```diff
@@ -93,7 +93,8 @@ <h1>Edit event</h1>
             <label for="tags">*Tags:</label>
           </div>
           <div class="form-input">
-            <!-- tag selection box will go here -->
+            <div class="tag-box">
+            </div>
```

Comment:
Just wondering, is there a specific reason the </div> is on a different line instead of <div></div>

Reply:
Nope, fixed!